labelbox/data/serialization/ndjson/classification.py

```diff
@@ -2,19 +2,20 @@
 
 from pydantic import BaseModel, Field, root_validator
 from labelbox.data.mixins import ConfidenceMixin
+from labelbox.data.serialization.ndjson.base import DataRow, NDAnnotation
 
 from labelbox.utils import camel_case
 from ...annotation_types.annotation import ClassificationAnnotation
 from ...annotation_types.video import VideoClassificationAnnotation
 from ...annotation_types.classification.classification import ClassificationAnswer, Dropdown, Text, Checklist, Radio
 from ...annotation_types.types import Cuid
 from ...annotation_types.data import TextData, VideoData, ImageData
-from .base import DataRow, NDAnnotation
 
 
 class NDFeature(ConfidenceMixin):
```

Comment:
Maybe it's more accurate to name this `NDBaseClassification`

Reply:
I did not have to go that route... turns out one of ND... subclasses already inherits from NDFeature and classification is there already

Reply:
I meant we should just rename this class, since it's confusing (I know you didn't name it). NDFeature makes it sound like it's a generic class for both objects and classifications, but it seems that its own use case right now is for classifications

Reply:
k got it now... I thought you wanted to like move it to a different file or smth